src/interop.rs

```diff
@@ -558,6 +566,10 @@ impl TryFrom<&tex::Entry> for Entry {
             item.set_abstract_(abstract_.into())
         }
 
+        if let Some(type_) = map_res(entry.type_())? {
+            item.set_genre(type_.into());
```

Comment:
I know this isn't really done elsewhere, but do you think you could add a comment here comparing the hayagriva and biblatex docs as you did in the PR description? The same could be done for the other change as well.